api/elections.go

```diff
@@ -623,37 +623,37 @@ func (a *API) electionFilterPaginatedHandler(msg *apirest.APIdata, ctx *httprout
 		Status         string         `json:"status"`
 	}{}
 	if err := json.Unmarshal(msg.Data, &body); err != nil {
-		return fmt.Errorf("cannot unmarshal request body: %w", err)
+		return ErrMarshalingServerJSONFailed.WithErr(err)
```

Comment:
this one is most definitely conceptually wrong, should be ErrCantParseDataAsJSON